superset/config.py

```diff
@@ -576,29 +577,43 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # Setup image size default is (300, 200, True)
 # IMG_SIZE = (300, 200, True)
 
-# Default cache timeout, applies to all cache backends unless specifically overridden in
+
+# Default cache config, applies to all cache backends unless specifically overridden in
 # each cache config.
-CACHE_DEFAULT_TIMEOUT = int(timedelta(days=1).total_seconds())
+def DEFAULT_CACHE_CONFIG_FUNC(  # pylint: disable=invalid-name
+    app: Flask,
+) -> CacheConfig:
+    default_timeout = app.config.get("CACHE_DEFAULT_TIMEOUT")
+    if default_timeout is None:
+        default_timeout = int(timedelta(days=1).total_seconds())
+    else:
+        logger.warning(
+            "The config flag CACHE_DEFAULT_TIMEOUT has been deprecated "
```

Comment:
This isn't quite true, i.e., this flag is defined by Flask-Caching per [here](https://flask-caching.readthedocs.io/en/latest/#configuring-flask-caching) but I get the gist of the comment.

Reply:
Ah yes - I was referring to the Superset config param, not the Flask-Caching cache config param. I agree and I see it now, this needs to be reworded.

Reply:
added "global" in front of "config flag" to distinguish from the Flask config.